com.unity.render-pipelines.universal/Runtime/Passes/DepthNormalOnlyPass.cs

```diff
@@ -69,14 +69,8 @@ public override void OnCameraSetup(CommandBuffer cmd, ref RenderingData renderin
         /// <inheritdoc/>
         public override void Execute(ScriptableRenderContext context, ref RenderingData renderingData)
         {
-            // NOTE: Do NOT mix ProfilingScope with named CommandBuffers i.e. CommandBufferPool.Get("name").
-            // Currently there's an issue which results in mismatched markers.
-            CommandBuffer cmd = CommandBufferPool.Get();
-            using (new ProfilingScope(cmd, ProfilingSampler.Get(URPProfileId.DepthNormalPrepass)))
+            using (new ProfilingScope(renderingData.commandBuffer, ProfilingSampler.Get(URPProfileId.DepthNormalPrepass)))
             {
-                context.ExecuteCommandBuffer(cmd);
-                cmd.Clear();
```

Comment:
Same here, you can't remove because there's a draw renderers being executed in the context that break the command buffer execution